TomEE-10.1/jre17/Semeru/ubuntu/microprofile/Dockerfile

```
@@ -56,13 +56,29 @@ RUN set -xe; \
     gpg --batch --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" ; \
   done
 
-ENV TOMEE_VER=10.0.1
+ENV TOMEE_VER=10.1.0
 ENV TOMEE_BUILD=microprofile
 
-RUN set -x \
-  && curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-${TOMEE_VER}/apache-tomee-${TOMEE_VER}-${TOMEE_BUILD}.tar.gz.asc -o tomee.tar.gz.asc \
-  && curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-${TOMEE_VER}/apache-tomee-${TOMEE_VER}-${TOMEE_BUILD}.tar.gz.sha512 -o tomee.tar.gz.sha512 \
-  && curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-${TOMEE_VER}/apache-tomee-${TOMEE_VER}-${TOMEE_BUILD}.tar.gz -o apache-tomee-${TOMEE_VER}-${TOMEE_BUILD}.tar.gz \
+RUN set -x; \
```

This should include at least `-e` as well (I'd recommend `-u` too, but that's a bigger bridge).

```suggestion
RUN set -ex; \
```